4.2.5/apache/gn-httpd.conf

```diff
@@ -0,0 +1,19 @@
+<VirtualHost *:80>
+  ServerName geonetwork
+
+  LogFormat "%v %h %l %u %t \"%r\" %>s %b \"%{Referer}i\" \"%{User-Agent}i\"" combined
+
+  ErrorLog /usr/local/apache2/logs/error.log
+  CustomLog /usr/local/apache2/logs/access.log combined
+
+  RedirectMatch ^/$ /geonetwork/
+  ProxyPass /geonetwork http://geonetwork:8080/geonetwork
+  ProxyPassReverse /geonetwork http://geonetwork:8080/geonetwork
```

Comment:
We need to add the WEBAPP_CONTEXT_PATH here

Reply:
Added here and to the nginx configuration